CRM/Civicase/Hook/NavigationMenu/AlterForCaseMenu.php

```diff
@@ -38,10 +39,12 @@ public function run(array &$menu) {
    *   Menu Array.
    */
   private function rewriteCaseUrls(array &$menu) {
+    $categoryId = CaseCategoryHelper::getCategoryByName('cases', 'id');
```

Comment:
Can you use the constant for the name here: `CRM_Civicase_Helper_CaseCategory::CASE_TYPE_CATEGORY_NAME`